examples/astro-instagram-demo/src/styles/global-loader.css

```diff
@@ -0,0 +1,3 @@
+/* using tailwindcss as a base for styles */
```

Comment:
you don't need this one in your example, and you should not use it in general (it was for stores before there was a proper design system in wix vibe)